spotinst/ocean_ecs_launch_specification/fields_spotinst_ocean_ecs_launch_specification.go

```diff
@@ -611,6 +611,7 @@ func Setup(fieldsMap map[commons.FieldName]*commons.GenericField) {
 					string(HTTPPutResponseHopLimit): {
 						Type:     schema.TypeInt,
 						Optional: true,
+						Default:  1357997531,
```

Comment:
this file doesn't need any change

Reply:
Made changes in this file to fix existing issue in ECS cluster creation, handled negative value passed by user to throw error and refrain resource update.